js/pages/characterizations/components/characterizations/characterization-view-edit/characterization-results.js

```diff
@@ -179,71 +179,90 @@ define([
-                                csvLine['P90'] = stat.p90;
-                                csvLine['StdDev'] = stat.stdDev;
-                                csvLine['Count'] = stat.count;
-                            }
-
-                            rAggr.push(csvLine);
-                            return rAggr;
-                        }, []);
-
-                        return [ ...aggr, ...lines ];
-                    }, []);
-                    CsvUtils.saveAsCsv(exprt);
-                },
-            }];
-
-            if (data.reports.length === 2 && type === 'prevalence') {
+        getButtonsConfig(type, data, analysis) {
+            const buttons = [];
+            if (type === 'prevalence') {
                 buttons.push({
-                    text: 'Export comparison',
-                    action: () => {
-                        const exprt = data.data.map(stat => {
-                            return {
-                                'Analysis ID': stat.analysisId,
-                                'Analysis name': stat.analysisName,  
-                                'Target cohort ID': stat.cohorts[0].cohortId,
-                                'Target cohort name': stat.cohorts[0].cohortName,
-                                'Comparator cohort ID': stat.cohorts[1].cohortId,
-                                'Comparator cohort name': stat.cohorts[1].cohortName,
-                                'Covariate ID': stat.covariateId,
-                                'Covariate name': stat.covariateName,
-                                'Target count': stat.sumValue[0],
-                                'Target prevalence or %': stat.pct[0],
-                                'Comparator count': stat.sumValue[1],
-                                'Comparator prevalence or %': stat.pct[1],
-                                'Std. Diff Of Mean': stat.stdDiff,
-                            };
-                        });
+                    text: 'Export prevalence to csv',
+                    action: ()  => {
+                        const exprt = data.reports.map((report) => {
+                            const lines = report.stats.reduce((rAggr, stat) => {
+                                const csvLine = {
+                                    'Analysis ID ': data.analysisId,
+                                    'Analysis name': data.analysisName,  
+                                    'Strata ID': stat.strataId,
+                                    'Strata name': stat.strataName,
+                                    'Cohort ID': report.cohortId,
+                                    'Cohort name': report.cohortName,
+                                    'Covariate ID': stat.covariateId,
+                                    'Covariate name': stat.covariateName,
+                                    'Count': stat.sumValue,
+                                    'Prevalence or %': stat.pct,
```

Comment:
Is this going to be prevalence or percentage? because the prevlance goes from 0 to 1 but percentage would go from 0 to 100.

If it comes out as .5, it's prevalence. If it comes out as 50%, it is percentage.